pkg/webhook/config.go

```diff
@@ -43,6 +43,8 @@ const (
 	// "fail", the init container will exit with error code 1. Defaults to "silent".
```

Comment:
Because I am currently testing it: What do you think about placing some logs when a container is excluded? for example: a container spins up which should be excluded and we log something along the line: "application container <name>  is being excluded"